### PR TITLE
Enable bazel test.

### PR DESCRIPTION
**Pull request by spxtr**
I'm skipping the python tests, they have an unmet dependency that I'll fix later. I removed the old redundant job.

### CODE REVIEW

File: jobs/pull-test-infra-bazel.sh [hunk 1]
```diff
@@ -20,6 +20,8 @@ set -o pipefail
 # Cache location.
 export TEST_TMPDIR="/root/.cache/bazel"
 
+find -L bazel-testlogs -name 'test.xml' -type f -exec rm '{}' +
```
Comment (ixdy) [inline jobs/pull-test-infra-bazel.sh]: maybe add a comment why you're doing this?

Comment (spxtr) [inline jobs/pull-test-infra-bazel.sh]: Done. I also had to put it after the build step, since the symlink won't exist in a fresh clone.

**Comment by spxtr** [comment]
OK, good to go. @ixdy PTAL

**Pull request merged**